tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -617,3 +617,23 @@ class resource_type(TypeInfo):
         def get(client, resource_info):
             return client.execute_command(
                 'get', {'project': resource_info['project_id']})
+
+
+@resources.register('dataproc-clusters')
```

Comment:
can we move this into a dataproc module, we generally segment service by module.

Reply:
done